Xamarin.Essentials/Calendars/Calendars.android.cs

```diff
@@ -424,8 +424,11 @@ static ContentValues SetupContentValues(CalendarEvent newEvent)
             eventValues.Put(CalendarContract.Events.InterfaceConsts.Dtend, newEvent.EndDate.HasValue ? newEvent.EndDate.Value.ToUnixTimeMilliseconds().ToString() : newEvent.StartDate.AddDays(1).ToUnixTimeMilliseconds().ToString());
             eventValues.Put(CalendarContract.Events.InterfaceConsts.EventTimezone, TimeZoneInfo.Local.Id);
             eventValues.Put(CalendarContract.Events.InterfaceConsts.EventEndTimezone, TimeZoneInfo.Local.Id);
-            eventValues.Put(CalendarContract.Events.InterfaceConsts.Deleted, 0);
```

Comment:
I think the point of this being set to 0 was to unset a previously applied Deleted flag (not sure exactly how that could happen). If we can't reproduce a scenario where we would want to unset the deleted flag, then I think this is fine.

Reply:
Have made it so that when you update existing events at present it will always set there deleted flag to false. Might not worry about it at a later stage, as it shouldn't be visible when it's deleted to begin with, but we may or may not change that. (maybe not for sample app, but maybe in the API call, have an optional parameter to show/hide deleted, haven't added this since it only affects android at this stage).